base/src/test/kotlin/proguard/analysis/GotoErrorTest.kt

```diff
@@ -0,0 +1,21 @@
+package proguard.analysis
+
+import io.kotest.assertions.throwables.shouldThrowAny
+import io.kotest.core.spec.style.FreeSpec
+import proguard.classfile.AccessConstants
+
+/**
+ * These test show errors that can occur with a `goto` instruction.
+ */
+class GotoErrorTest : FreeSpec({
```

Comment:
This one is for the `CompactCodeAttributeComposer`?

Reply:
Right! Do I omit it from this PR?